src/main/java/org/springframework/data/jpa/repository/support/CrudMethodMetadata.java

```diff
@@ -45,10 +47,18 @@
 	Map<String, Object> getQueryHints();
 
 	/**
-	 * Returns the {@link JpaEntityGraph} to be used.
+	 * Returns the {@link EntityGraph} to be used.
 	 * 
 	 * @return
-	 * @since 1.8
+	 * @since 1.9
 	 */
-	JpaEntityGraph getEntityGraph();
+	EntityGraph getEntityGraph();
+	
+	/**
+	 * Returns the {@link Method} to be used.
+	 * 
+	 * @return
+	 * @since 1.9
+	 */
+	Method getMethod();
```

Comment:
This one feels weird, as the sole purpose of the `CrudMethodMetdata` is for clients not having to deal with the method being invoked. See more comments on that on the spot in `SimpleJpaRepository` where this is used.